### ISSUE TITLE
Bug Summary

### ISSUE THREAD

**Issue by habemus-papadum** [fpgmaas/cookiecutter-poetry#93]
**Describe the bug**

`make docs` fails on a new project with:

```
Error reading page 'modules.md': No module named 'mkdocstrings.handlers.python'
```

**Comment by fpgmaas** [comment]
Thanks for raising this issue! I tried to reproduce it by running the following commands:

```
 2762  pip install cookiecutter
 2763  cookiecutter https://github.com/fpgmaas/cookiecutter-poetry.git
 2764  cd example-project
 2765  poetry install
 2766  make docs
 ```
 
 This seemed to work fine for me. Could you maybe add some more details on how to reproduce your issue?

**Comment by habemus-papadum** [comment]
Hi there -- 
I was able to work around the issue by bumping the constraints on `mkdocstrings` in the  `pyproject.yaml`:

```
mkdocstrings = {version = "^0.22.0", extras = ["python"]}
mkdocstrings-python = "^1.7.0"
```
(My values may be overly conservative).  In my case, I was getting a `mkdocstrings-python` < 1.0 with the current settings in `ccp` .

Also, to get the pages to deploy to gh-pages on release, I needed to tweak the permissions in the release workflow yaml (again, I probably used settings that are too broad).
my repo is here: https://github.com/habemus-papadum/how-now if you want to see the exact settings.  

I'm guessing it is due to me using python3.11.  But I feel this issue is resolved on my end so am going to mark it closed. 

Thank you for create `CCP`!
